fastlane/Fastfile

```diff
@@ -240,6 +273,25 @@ platform :android do
     )
   end
 
+  lane :build_qa do
+    prepare_release
```

Comment:
why aren't you making sure to delete the existing `./Mattermost.apk` file as you do with the ipa file?

Reply:
Removed delete of ipa.